x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/boosts/boost_item_content/boost_item_content.scss

```diff
@@ -0,0 +1,6 @@
+.boosts {
+  .boosts__itemContent {
+    border: none;
+    border-top: $euiBorderThin;
+  }
+}
```

Comment:
`relevanceTuningForm__itemContent` is kind of like ... a semantically different element than `boosts__itemContent`, so I'm hesitant to do that.

Reply:
In this case the styling isn't semantic however - it's simply a common border style. Can I suggest a separate class name in that case that makes it clear what the intention of the styling is? Something like `.relevanceTuningAccordionItem` perhaps?

Reply:
Yeah that make sense. Will update.

Reply:
Good call, that removed to scss files. I also added a border-radius of 0 because instead of a straight line it was slightly curved, which is not what I wanted

c31c670f54a6d89cfdfded5b78a124eba057776c